src/operations.ts

```diff
@@ -1,6 +1,7 @@
 import { Resolver } from 'did-resolver'
 import { getResolver } from 'ethr-did-resolver'
-import { verifyJWT, decodeJWT } from 'jesse-did-jwt'
+import { decodeJWT } from 'did-jwt'
+import { verifyJWT } from 'jesse-did-jwt'
```

Comment:
I'm getting _Module not found: Can't resolve 'did-jwt' in '/Users/ilanolkies/code/rif-identity/rif-credential-verifier/src'_

```suggestion
import { decodeJWT, verifyJWT } from 'jesse-did-jwt'
```

Reply:
Thanks for catching this. They have been combined.